### ISSUE TITLE
webextensions.api.tabs.update - updateProperties `loadReplace` and `successorTabId` are not supported by Chrome

### ISSUE THREAD

**Issue by Marcono1234** [mdn/browser-compat-data#15412]
#### What information was incorrect, unhelpful, or incomplete?

It looks like Chrome does not support all properties of [`updateProperties`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/update#updateproperties), in particular [Chrome's documentation](https://developer.chrome.com/docs/extensions/reference/tabs/#type-update-updateProperties) does not list the following:
- `loadReplace`
- `successorTabId`

#### What did you expect to see?

The compatibility table should indicate that Chrome support for `updateProperties` is only partial.

#### Did you test this? If so, how?

For a custom extension using https://github.com/mozilla/webextension-polyfill I encountered an exception when trying to use `loadReplace: true` for Chrome Version 99.0.4844.74 (Official Build) (64-bit) on Windows 10:
```
TypeError: Error in invocation of tabs.update(optional integer tabId, object updateProperties, optional function callback): Error at parameter 'updateProperties': Unexpected property: 'loadReplace'.
    at browser-polyfill.js:859:27
    at new Promise (<anonymous>)
    at Proxy.asyncFunctionWrapper (browser-polyfill.js:836:18)
    at Object.apply (browser-polyfill.js:891:28)
    at background.ts:88:8
    at onMessage (browser-polyfill.js:1120:22)
(anonymous) @ background.ts:94
```

<details>
<summary>MDN page report details</summary>

* Query: `webextensions.api.tabs.update`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/update
* Report started: 2022-03-17T20:48:27.437Z

</details>

**Comment by queengooborg** [comment]
Our data actually indicates that there's no support for either of the two properties in Chrome, but it's not well represented on the MDN page.  We'll reorganize the compat data to better represent this!

**Comment by Marcono1234** [comment]
Thanks! Will this be fixed by #15616?

**Comment by queengooborg** [comment]
That's correct!
